make_profiler/lint_makefile.py

```
@@ -174,6 +205,14 @@ def validate(
     return is_valid
 
 
+def summarize_errors(errors: List[LintError]) -> str:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)